# HG changeset patch
# User Rik <rik@octave.org>
# Date 1409030342 25200
#      Mon Aug 25 22:19:02 2014 -0700
# Node ID 3d0f4f4ec688c6d41b3a99cc5d27b2c9f3fa3f5b
# Parent  2f117c4b5cb0ca9a6353377a52ad129ed0b497a5
NEWS: note that isprime now accepts negative and complex inputs.

NEWS: note that isprime now accepts negative and complex inputs.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -53,16 +53,18 @@ Summary of important user-visible change
     radial rho axis rather than using a rectangular x/y axis.
 
  ** linkprop has been completely re-coded for performance and Matlab
     compatibility.  It now returns a linkprop object which must be stored
     in a variable for as long as the graphic objects should remain linked.
     To unlink properties use 'clear hlink' where hlink is the variable
     containing the linkprop object.
 
+ ** isprime has been extended to operate on negative and complex inputs.
+
  ** The new warning ID "Octave:data-file-in-path" replaces the three
     previous separate warning IDs "Octave:fopen-file-in-path",
     "Octave:load-file-in-path", and "Octave:md5sum-file-in-path".
 
  ** Other new functions added in 4.2:
 
       bandwidth            isbanded        javachk
       dir_in_loadpath      isdiag          linkaxes
